public/data/myanmar/pop/pop_2022-07-21.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Let's use underscore in file names when possible -> `2022_07_21`

Reply:
removed the myanmar test data and updated rbd to use underscores